fire/cli/niv/_regn.py

```diff
@@ -195,14 +189,14 @@ def gama_beregning(
 
         # Observationer
         gamafil.write("<height-differences>\n")
-        for obs in observationer.itertuples(index=False):
-            if obs.Sluk == "x":
+        for obs in observationer:
+            if obs[1] == "x":
                 continue
             gamafil.write(
-                f"<dh from='{obs.Fra}' to='{obs.Til}' "
-                f"val='{obs.ΔH:+.6f}' "
-                f"dist='{obs.L:.5f}' stdev='{spredning(obs.Type, obs.L, obs.Opst, obs.σ, obs.δ):.5f}' "
-                f"extern='{obs.Journal}'/>\n"
+                f"<dh from='{obs[2]}' to='{obs[3]}' "
+                f"val='{obs[4]:+.6f}' "
+                f"dist='{obs[5]:.5f}' stdev='{spredning(obs[17], obs[5], obs[6], obs[7], obs[8]):.5f}' "
+                f"extern='{obs[0]}'/>\n"
```

Comment:
Her er et meget godt eksempel på et sted det giver mening at lave en bedre refaktorisering. Vi har 18 (måske flere?) informationer om en observation men bruger tilsyneladende kun cirka 10 af dem. `skriv_gama`\`gama_udjævn` bør kun tage det information som det har brug for som input, den skal ikke forholde sig til alt muligt andet info der er relevant for den kaldende kode.

Det betyder så nok også at der i `regn` skal laves lidt ekstra arbejde for at pille relevant information ud af pandas dataframes og efterfølgende arbejde for at opdatere dataframes igen efter udjævningen.